frontend/src/pages/Home/Home.jsx

```diff
@@ -5,19 +5,17 @@ import "./NavigationButton.scss";
 
 function Home() {
   return (
-    <>
+    <section className="buttons-home">
       <img className="logo" src={GeoCode} alt="logo de l'application" />
-      <section className="buttons-home">
-        <NavigationButton destination="/subscribe" text="s'enregistrer" />
-        <NavigationButton destination="/connection" text="Se connecter" />
-        <p className="consul-s-inscription">
-          <NavigationButton
-            destination="/map"
-            text="Consulter sans inscription"
-          />
-        </p>
-      </section>
-    </>
+      <NavigationButton destination="/subscribe" text="s'enregistrer" />
+      <NavigationButton destination="/connection" text="Se connecter" />
+      <p className="consul-s-inscription">
```

Comment:
Why a tag <p> ?

Reply:
I have no idea, I don't remember doing this : for this component I just intented to remove a useless container, and to make some minor changes in CSS that affected all other components (background color applied on "body")
Amended !